doc/contrib/documentation.md

```diff
@@ -0,0 +1,84 @@
+# Documentation
+
+This document gives guidelines for contributors concerning Elektra's documentation.
+This document takes preference to the actual situation, so if you see documentation
+not according to this document, please [create an issue](https://issue.libelektra.org)
+or directly fix it with your next PR.
```

Comment:
I prefer writing docs with one sentence per line and using the line wrap feature of my editor. For the rendered results it doesn't matter, but with one sentence per line future git diffs are simpler. There is no noisy diff, because the line wrapping changed. It also makes Code Review suggestions on GitHub easier.

If we want documentation written that way, we should probably mention it in this file.

Reply:
Yes, I agree and fixed the document but for different reasons. I actually think diffs tend to be better with more line breaks (line wrapping is rarely changed by anyone). But: using single lines per sentences make the sentence length better visible and so can lead to shorter and better to read sentences, and this is a more important goal than nice diffs and may avoid sentences like this one.